README.md

```diff
@@ -113,16 +113,27 @@ Libraries for publishing to NPM should have their directory names prefixed with
 
 Apps should have their directory names prefixed with `app-`, e.g. `/project` becomes `/app-project`.
 
-## Production deployment
+## Deployment
 
-Deploys to production and staging are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image (`zooniverse/front-end-monorepo`) is built with the top-level Dockerfile, which installs and builds production versions of the `lib-` packages. 
+Deploys to production and staging are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/) using Docker images (descriptions found below). 
```

Comment:
'Docker images' should be a link to the Docker images heading.

Reply:
I gave a brief step-by-step description of the Jenkins job in #1123. It's all been removed here but should we keep it in? I thought it was useful to clarify in my mind what Jenkins does when it's deploying the apps.

Reply:
I copy and pasted those lines into the docker images section.